packages/react/src/components/Button/Button.stories.mdx

```diff
@@ -34,9 +35,12 @@ import {ArgsTable} from "@storybook/addon-docs";
         disabled: {
             control: { type: 'boolean'}
         },
+        ariaDisabled: {
```

Comment:
I think this can changed to `"aria-disabled"` as the native attribute? Just to avoid confusion so people don't think its an actual prop on the `Button` now :)

Reply:
I agree, and I tried to do that, but I couldn't get it to work.
When I used `'aria-disabled'` (although with ' instead of ") I just got a bunch of errors... 😕

Reply:
I agree that it may be confusing, but the props right name does show up in the right way in the 'show code'-feature though. Which is the way I would prefer to see its use anyway. The only place you see ariaDisabled is in the constrol section of the canvas.
![image](https://user-images.githubusercontent.com/22012997/225257740-c1411d9a-d1c2-4efa-8a5e-607bd5f7516f.png)

Reply:
> I agree, and I tried to do that, but I couldn't get it to work. When I used `'aria-disabled'` (although with ' instead of ") I just got a bunch of errors... 😕

Weird 🤔 Storybook is pretty "special" sometimes though 🙈